actors/builtin/multisig/multisig_actor.go

```diff
@@ -32,6 +34,15 @@ type Transaction struct {
 	Approved []addr.Address
 }
 
+type ApprovalHashData struct {
```

Comment:
naming inconsistency:
this is called `ApprovalHashData` but its hash in `TxnIDParams` is call `ProposalHash`, and its contents is the `ProposeParams` (+ `MsigWallet` and `Requester`)